packages/postgres-database/src/simcore_postgres_database/models/projects_nodes.py

```diff
@@ -38,27 +38,135 @@
         ),
         nullable=False,
         index=True,
-        doc="The project unique identifier",
+        comment="Unique identifier of the project",
     ),
     sa.Column(
         "node_id",
         sa.String,
         nullable=False,
         index=True,
-        doc="The node unique identifier",
+        comment="Unique identifier of the node",
     ),
     sa.Column(
         "required_resources",
         JSONB,
         nullable=False,
         server_default=sa.text("'{}'::jsonb"),
-        doc="The node required resources",
+        comment="Required resources",
     ),
     # TIME STAMPS ----
     column_created_datetime(timezone=True),
     column_modified_datetime(timezone=True),
+    sa.Column(
+        "key",
+        sa.String,
+        nullable=False,
+        comment="Distinctive name (based on the Docker registry path)",
+    ),
+    sa.Column(
+        "version",
+        sa.String,
+        nullable=False,
+        comment="Semantic version number",
+    ),
+    sa.Column(
+        "label",
+        sa.String,
+        nullable=False,
+        comment="Short name ",
+    ),
+    sa.Column(
+        "progress",
+        sa.Numeric,
+        nullable=True,
+        comment="Progress value",
+    ),
+    sa.Column(
+        "thumbnail",
+        sa.String,
+        nullable=True,
+        comment="Url of the latest screenshot",
+    ),
+    sa.Column(
+        "input_access",
+        JSONB,
+        nullable=True,
+        comment="Map with key - access level pairs",
+    ),
+    sa.Column(
+        "input_nodes",
+        JSONB,  # Array
+        nullable=True,
+        comment="IDs of the nodes where is connected to",
+    ),
+    sa.Column(
+        "inputs",
+        JSONB,
+        nullable=True,
+        comment="Input properties values",
+    ),
+    sa.Column(
+        "inputs_required",
+        JSONB,  # Array
```

Comment:
it seems postgres also understand array of booleans... maybe worth getting rid of most of these undefined JSON types... that is the idea of moving to its own table after all